about.html

```diff
@@ -0,0 +1,349 @@
+            <img
+              class="Event-resources-2"
+              src="resources/bassibhai.jpg"
+              alt="Anubhav Show"
+            />
+            <div class="contact-year-text">
+              <h3 class="contact-year-text-head font">2023</h3>
+              <h4 class="contact-year-text-info font">
+                ComedyBurst Event 2023 Pune
+              </h4>
+            </div>
+          </section>
+        </div>
+        <div class="last-Event-container for-desktop">
+          <section class="last-Event-item for-desktop">
+            <div
+              class="Event-year-location Event-cover-transition"
+            ></div>
+            <img
+              class="Event-resources-1"
+              src="resources/zakirbhai.jpg"
+              alt="zakri bhai show"
+            />
+            <div class="contact-year-text">
+              <h3 class="contact-year-text-head font">2023</h3>
+              <h4 class="contact-year-text-info font">
+                SongKick Event 2023 Mumbai
+              </h4>
+            </div>
+          </section>
+          <section class="last-Event-item for-desktop">
+            <div
+              class="Event-year-location-2 Event-cover-transition"
+            ></div>
+            <img
+              class="Event-resources-2"
+              src="resources/bassibhai.jpg"
+              alt="bassi show"
+            />
+            <div class="contact-year-text">
+              <h3 class="contact-year-text-head font">2023</h3>
+              <h4 class="contact-year-text-info font">
+                ComedyBurst1 Event 2023 Pune
+              </h4>
+            </div>
+          </section>
+        </div>
+      </section>
+      <footer>
+        <section class="partners">
+          <h3 class="partner-head for-phone font">Partner</h3>
+          <div class="partner-head-red-line for-phone font">________</div>
+          <ul class="partners-logo-ul for-phone">
+            <li>
+              <img
+                class="partner-logo for-phone"
+                src="resources/icons/bookmyshow.png"
+                alt="bookmyshow Logo"
+                role="presentation"
+              />
+            </li>
+            <li>
+              <img
+                class="partner-logo for-phone"
+                src="resources/icons/swiggy.png"
+                alt="swiggy Logo"
+                role="presentation"
+              />
+            </li>
+            <li>
+              <img
+                class="partner-logo for-phone"
+                src="resources/icons/zomato.png"
+                alt="zomato Logo"
+                role="presentation"
+              />
+            </li>
+          </ul>
+          <div class="footer-end for-desktop">
+            <section class="footer-end-items-desk for-desktop">
+              <h1 class="footer-end-items-logo-desk font for-desktop">
+                ©️ComedyBurst
+              </h1>
```

Comment:
- [ ] Kindly note that the `h1` tag should be used only once on each page, kindly use another `hx`.

Reply:
Done 👍 @Ofarouq310